message_definitions/v1.0/ras_a.xml

```diff
@@ -381,7 +381,6 @@
       <field type="uint8_t" name="confidence_classification" units="%" invalid="UINT8_MAX">Confidence level of the POI classification. 0..100, UINT8_MAX when unknown.</field>
       <field type="uint8_t" name="confidence_localization" units="%" invalid="UINT8_MAX">Confidence level of the POI localization. 0..100, UINT8_MAX when unknown.</field>
       <field type="uint16_t" name="ttl" units="s">Time to live: If this time has elapsed since last update, the POI should be deleted on the receiver side. A value of 0 should indicate no timeout.</field>
-      <field type="uint8_t" name="status_flags">Bitmask for POI status. Bit 1: POI is in focus on camera, Bit 8: POI has been cleared and should be deleted.</field>
```

Comment:
@TSC21 @thomas-watters-skydio We're currently using `status_flags` as a way to indicate _immediate_ (i.e., as opposed to _eventual_ e.g., TTL) removal/deletion of a POI.  Unless we have an alternative approach for immediate deletion, we need to keep this.